recipes/libpng/all/conanfile.py

```diff
@@ -1,5 +1,10 @@
-from conans import ConanFile, CMake, tools
 import os
+from conan import ConanFile
+from conans import tools, CMake
+try:
+    from conan.tools.cross_building import cross_building
+except ImportError:
+    from conan.tools.build.cross_building import cross_building
 
 required_conan_version = ">=1.43.0"
```

Comment:
```suggestion
from conan.tools.build.cross_building import cross_building

required_conan_version = ">=1.45.0"
```
Move forward to Conan 2.0: https://docs.conan.io/en/latest/migrating_to_2.0